docs/userGuide/syntax/diagrams.md

```diff
@@ -142,6 +142,11 @@ name   | `string` | The name of the output file.<br>Avoid using the same name fo
 src    | `string` | The URL of the diagram if your diagram is in another `.puml` file.<br>The URL can be specified as absolute or relative references. More info in: _[Intra-Site Links]({{baseUrl}}/userGuide/formattingContents.html#intraSiteLinks)_
 width  | `string` | The width of the diagram in pixels.<br>If both width and height are specified, width takes priority over height. It is to maintain the diagram's aspect ratio.
 
+<box type="tip">
+
+It's also possible to utilize JavaScript diagram libraries such as [Mermaid](https://mermaid-js.github.io/mermaid/) via the [Mermaid plugin](../usingPlugins.html#plugin-mermaid).
```

Comment:
Hmm this seems to be implying we support more than one library HAHHA 

Also can we use the baseUrl convention here!

Reply:
As we now have two diagram types, we can put them under separate sub-headings?

e.g.,

>## Diagrams
>### PlantUML Diagrams
>### Mermaid.js Diagrams

Reply:
Can you address these comments @Tim-Siu ?

Reply:
I dont think we need this box anymore now the mermaid is included in the diagram part of the UG

Reply:
Hi @Tim-Siu can you address this comment

Reply:
> Hi @Tim-Siu can you address this comment

I think this has already been resolved.